lib/kubernetes-deploy/runner_task.rb

```diff
@@ -45,14 +45,17 @@ def run!(task_template:, entrypoint:, args:, env_vars: [], verify_result: true)
       else
         record_status_once(pod)
       end
-      ::StatsD.distribution('task_runner.duration', StatsD.duration(start), tags: statsd_tags('success'))
+      StatsD.distribution('task_runner.duration', StatsD.duration(start),
+        tags: statsd_tags('success'), prefix: "KubernetesDeploy")
       @logger.print_summary(:success)
     rescue DeploymentTimeoutError
-      ::StatsD.distribution('task_runner.duration', StatsD.duration(start), tags: statsd_tags('timeout'))
+      StatsD.distribution('task_runner.duration', StatsD.duration(start),
+        tags: statsd_tags('timeout'), prefix: "KubernetesDeploy")
       @logger.print_summary(:timed_out)
       raise
     rescue FatalDeploymentError
-      ::StatsD.distribution('task_runner.duration', StatsD.duration(start), tags: statsd_tags('failure'))
+      StatsD.distribution('task_runner.duration', StatsD.duration(start),
```

Comment:
could we overwrite this (and the other similar methods) in our class to call `super` with the prefix so that we don't have to write out the prefix every time?

Reply:
I was thinking the same thing and agree with you. I'll make the change